adapters/richaudience/richaudience.go

```diff
@@ -26,36 +26,14 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters
 }
 
 func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	raiHeaders := http.Header{}
-
-	setHeaders(&raiHeaders)
-
 	isUrlSecure := getIsUrlSecure(request)
```

Comment:
You don't seem to use this except to pass to `setImp()`, and `setImp()` is already being passed `request`. Any reason to not move it into this function?

Reply:
I have resolved this in next commit, thanks for your comments